cypress/integration/Page/publicAuctionFilterPage.js

```diff
@@ -88,15 +41,8 @@ describe('Testcase for Manheim filter', () => {
                 })
             })
         })
-    })
-    it('Verify conditon information filter', () => {
-        cy.visit('/')
-        cy.get('.uhf-user__locations').click()
-        cy.contains('U.S. Locations').click()
-        cy.contains('Manheim Cincinnati').click()
-        cy.contains('Public Auctions').click()
-        cy.contains('See Inventory').click()
-        cy.wait(10000)
+    }
+    verifyConditionInformation(){
         cy.get('[data-test-id="filter: condition information"]').then(condition => {
             cy.wrap(condition).click()
             cy.get('[data-test-id="With Condition Report"]').check()
```

Comment:
Should separate the step choose the filter and the step verify to 2 the functions. The first one is the function `chooseFilterCondition()` and the second one is the function `validateFilter()`.
Please check and update the code in other places.

Reply:
Any update for this comment @LamAnLeTran @namthtran98 ?

Reply:
i've update the first and the last comment, i'm trying to update the second comment and this comment i haven't updated